products/facilities/facdb/datasets.yml

```diff
@@ -156,9 +156,9 @@ datasets:
     scripts:
       - dpr_parksproperties.sql
 
-  - name: dycd_afterschoolprograms
+  - name: dycd_service_sites
     scripts:
-      -  dycd_afterschoolprograms.sql
+      -  dycd_service_sites.sql
```

Comment:
From whatever happens with this file, would it be easy to scrap and just use the recipe instead? Just curious

Reply:
seems like FacDB has a sql script like this for every source. it standardizes column names, does some categorizing, and makes a geometry column from lat long columns. and at the end it does `CALL append_to_facdb_base('_dycd_service_sites');`

so scrapping this one would probably scrapping and combining em all

Reply:
Gotcha. Or logic to just call script based on dataset name without using this file?

Reply:
oh did you mean scrapping the `datesets.yml` file? def agree this file isn't the best way to do this!

Reply:
Precisely!